docs/ml-features.md

```diff
@@ -114,7 +116,10 @@ for more details on the API.
  During the fitting process, `CountVectorizer` will select the top `vocabSize` words ordered by
  term frequency across the corpus. An optional parameter "minDF" also affect the fitting process
  by specifying the minimum number (or fraction if < 1.0) of documents a term must appear in to be
- included in the vocabulary.
+ included in the vocabulary.Another optional binary toggle parameter controls the output vector.
```

Comment:
I said "Another", because the previous line starts with 'An optional parameter". It just sounded right